### PR TITLE
test(i): Add edge cases for collection delete command

### LINKED ISSUE #4802: Delete collection suger syntax edge cases to test
Delete collection suger syntax edge cases to test

### PR DESCRIPTION
**Pull request by shahzadlone**
## Relevant issue(s)
- Resolves #4802

## Description
Adds breadth coverage for `DeleteCollection` edge cases that I was worried about.

### CODE REVIEW

File: cli/collection_delete.go [hunk 1]
```diff
@@ -40,13 +40,9 @@ The named collections must not contain any documents. Delete all documents first
...
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var names []string
-			for name := range strings.SplitSeq(args[0], ",") {
-				name = strings.TrimSpace(name)
-				if name == "" {
-					continue
-				}
-				names = append(names, name)
+			names := strings.Split(args[0], ",")
```
Comment (shahzadlone) [inline cli/collection_delete.go:43]: info: Fixed this by not doing validation outside client, otherwise cli validation would differ from other clients.

**Review by AndrewSisley — APPROVED** [review]
LGTM, but please check the rabbit comments if you haven't before merge, they sound reasonable, but I haven't look very closely :)
